flow-node

```diff
@@ -84,6 +84,10 @@ if (options.length > 0) {
 
 require('./register')({ all: all });
 
+function withoutOptions(optionsToExclude, options) {
+  return options.filter((option) => !optionsToExclude.includes(option))
```

Comment:
Seeing that Travis runs tests all the way down to Node v0.10, I guess that answers my previous question..